Management/src/ActuatorApi/BasicAuthExtensions.cs

```diff
@@ -0,0 +1,35 @@
+using idunno.Authentication.Basic;
+using System.Security.Claims;
+
+namespace Steeltoe.Samples.ActuatorApi;
+
+public static class BasicAuthExtensions
+{
+    public static void ConfigureActuatorAuth(this IServiceCollection serviceCollection)
+    {
+        var builder = serviceCollection.AddAuthentication(BasicAuthenticationDefaults.AuthenticationScheme);
+        builder.AddBasic(BasicAuthenticationDefaults.AuthenticationScheme, options =>
+        {
+            options.ForwardDefaultSelector = httpContext => httpContext.Request.Path.StartsWithSegments("/actuator") ? BasicAuthenticationDefaults.AuthenticationScheme : null;
+            options.ForwardChallenge = BasicAuthenticationDefaults.AuthenticationScheme;
+
+            options.Events = new BasicAuthenticationEvents
+            {
+                OnValidateCredentials = (validateCredentialsContext) =>
+                {
+                    // HardCoded as an example, do not use any of this in Production!
```

Comment:
```suggestion
                    // This sample hard-codes the username and password for simplicity. In a real-world scenario, they are typically fetched from an external system.
```

Reply:
I expect this and the OpenTelemetry extension found in both apps in the management sample to move to ServiceDefaults in #344

Reply:
It's unclear to me how that would work on Cloud Foundry. Switching to Aspire requires Steeltoe to support Eureka-on-Aspire first. Either way, we'll still need it until #344 is implemented.